src/include/tcop/tcop.h

```diff
@@ -44,6 +44,10 @@ class TrafficCop {
   // static singleton method used by tests
   static TrafficCop &GetInstance();
 
+  // # 623
+  //  concurrency::Transaction *consistentTxn;
+  bool single_statement_txn;
```

Comment:
also, member variables should be named as `single_statement_txn_`, rather than `single_statement_txn`.